pytorch_blade/torch_blade/mlir/hash.py

```diff
@@ -31,10 +26,10 @@ def get_graph_hash(graph):
     for input in graph.inputs():
         val_info = _backends.TensorInfo(input)
         rank = len(val_info.sizes)
-        hash_value = hash_combine(hash_value, data_hash(val_info.dtype))
-        hash_value = hash_combine(hash_value, data_hash(val_info.device))
-        hash_value = hash_combine(hash_value, data_hash(str(rank)))
+        hash_value = hash_combine(hash_value, hash_data(val_info.dtype))
+        hash_value = hash_combine(hash_value, hash_data(val_info.device))
+        hash_value = hash_combine(hash_value, hash_data(str(rank)))
     # hash node info
     for n in nodes:
-        hash_value = hash_combine(hash_value, data_hash(n.kind()))
+        hash_value = hash_combine(hash_value, hash_data(n.kind()))
```

Comment:
Can we add input dtype of nodes? Some models in TorchBench meet this problem that graph input is int, and model are half or float
```
        for input in n.inputs():
                hash_value = hash_combine(hash_value, hash_data(input.type().str()))
```

Reply:
That's a good point! Thanks.